lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -6053,7 +6053,7 @@ LLVMToSPIRVBase::transBuiltinToInstWithoutDecoration(Op OC, CallInst *CI,
     OC = OpAtomicCompareExchange;
   if (isGroupOpCode(OC))
     BM->addCapability(CapabilityGroups);
-  switch (OC) {
+  switch (static_cast<size_t>(OC)) {
```

Comment:
Why is this static_cast necessary?

Reply:
Because of the added `case internal::` - such OpCodes are not a part of `spv::Op` enum, so this trick helps to fix the build warning. It's used not only here but in all switch cases with internal opcodes in SPIRV writer part.